tensorflow_lite_support/python/task/text/pybinds/_pywrap_text_searcher.cc

```diff
@@ -57,19 +55,9 @@ PYBIND11_MODULE(_pywrap_text_searcher, m) {
 
             std::unique_ptr<CppSearchOptions> cpp_search_options =
                 std::make_unique<CppSearchOptions>();
-            if (search_options.has_index_file_content()) {
-              cpp_search_options->mutable_index_file()->set_file_content(
-                  search_options.index_file_content());
-            }
-            if (search_options.has_index_file_name()) {
-              cpp_search_options->mutable_index_file()->set_file_name(
-                  search_options.index_file_name());
-            }
-            if (search_options.has_max_results()) {
-              cpp_search_options->set_max_results(search_options.max_results());
-            }
-
+            cpp_search_options->CopyFrom(search_options);
```

Comment:
nit: Can you use the search_options directly instead of having to copy it?

Reply:
@khanhlvg I can change it, however, I believe in this [commit](https://github.com/tensorflow/tflite-support/commit/e109aa0094e6fd5bb1e30c61ec9b19194c452ac3), we decided to set the options by copying it from the source. WDYT?

Reply:
You're right. Copying is needed from the memory management perspective.